docs/artifactory/mvn/help.go

```diff
@@ -2,13 +2,10 @@ package mvn
 
 const Description = "Run Maven build."
 
-var Usage = []string{`jfrog rt mvn "<goals and options>" <config file path> [command options]`, `jfrog rt mvn "<goals and options> -f path/to/pom.xml" <config file path> [command options]`}
+var Usage = []string{`jfrog rt mvn "<goals and options>" [command options]`, `jfrog rt mvn "<goals and options> -f path/to/pom.xml" [command options]`}
```

Comment:
The usage still shows that the maven arguments are sent as a single argiment:
"[goals and options] -f path/to/pom.xml" --> [goals and options]
and 
"[goals and options]" --> [goals and options]

The same goes for the gradle

Reply:
Fixed